tests-web/www/js/layers.js

```diff
@@ -1,49 +1,70 @@
 export const makePlainGeoJsonLayer = (text) => {
-  return new L.geoJSON(JSON.parse(text), { style: styleGeoJson });
-};
+  const intersectionColours = {
```

Comment:
It felt a bit cleaner to me to group everything related to one layer in the same function, but happy to reconsider